docs/quickstart.mdx

```diff
@@ -35,14 +35,21 @@ Our test will succeed if the model outputs valid JSON.
   </Step>
 
   <Step title="Run the test">
+    This step requires an `OPENAI_API_KEY` to authenticate with OpenAI.
+    
+    Export `OPENAI_API_KEY` as an environment variable.
+
+    ```sh
+    export "OPENAI_API_KEY=your_api_key_here"
```

Comment:
a better shorthand could be `OPEN_API_KEY=your_api_key_here npx @empiricalrun/cli run` ?

Reply:
I was thinking on the same lines as you mentioned, keeping the CLI command cleaner (`npx @empiricalrun/cli run`). It can be like a sub-step in this step. 

We'll also add a link to setup `.env` too below.